framework/db/mapping_manager.py

```diff
@@ -74,6 +74,14 @@ def GetMappingTypes(self):
         """
         return self.mapping_types
 
+    def get_all_mappings(self):
+        """
+        Returns a dictionary with key: owtf_code and value: mapping dictionary.
```

Comment:
Could you try to follow Sphinx's syntax? You can find an example at https://github.com/owtf/ptp/blob/develop/ptp/libptp/parser.py#L57

A suggestion: would you find the following docstring clearer?
```python
def get_all_mappings(self):
    """Create a mapping between OWTF plugins code and OWTF plugins description.

    :return: Mapping dictionary {code: description, code2: description2, ...}
    :rtype: dict
    """
```

@saganshul Thanks for keeping the code base documented btw :) really appreciated!